src/game/bg_misc.c

```diff
@@ -2532,7 +2532,7 @@ qboolean BG_PlayerTouchesItem(playerState_t *ps, entityState_t *item, int atTime
 
 	BG_EvaluateTrajectory(&item->pos, atTime, origin, qfalse, item->effect2Time);
 
-	return (vec3_distance(ps->origin, origin) < 36);
+	return (vec3_distance(ps->origin, origin) < (ps->maxs[0] - ps->mins[0]));
```

Comment:
wouldn't using X value only cause some pickups items that have larger bbox X to be picked up easier than others?
Wouldn't it be more accurate to rather trace the bounding box?

Reply:
`ps->mins/maxs` are player boundaries, not item boundaries. The old code just simply hardcoded 36 as the value (`18 + -18`, which is the same as `ps->maxs[0] - ps->mins[0]`).

Reply:
oh so its explained in the in the referenced issue, but still this doesn's seem like proper solution not that the solution was explained or tested

Reply:
oh I see missed that its from ps, thanks